sass/base/components/navigation/main-nav.twig

```diff
@@ -1,34 +1,82 @@
-<header role="banner">
-  <div class="region-header-inner-wrapper">
-    <div class="region region-header">
-      <a href="/" title="Home" rel="home" class="site-logo">
-        <img src="/profiles/contrib/cr/themes/custom/campaign_base/logo.svg" alt="Home">
-      </a>
-      <div class="main-menu__icons">
-        <button class="c-hamburger c-hamburger--htx main-menu-toggle" title="open here">
-          <span>menu</span>
-        </button>
+<div class="header__inner-wrapper">
```

Comment:
I guess we're just changing the `header` element to a `div` for the styleguide only, right?